test/components/previews/forms/toggle_component_preview.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Forms
+  class ToggleComponentPreview < ViewComponent::Preview
+    include ActionView::Helpers::FormHelper
+    attr_accessor :output_buffer
+
+    def default
+      f = ActionView::Helpers::FormBuilder.new(:form, nil, self, {})
+      render(Forms::ToggleComponent.new(form: f, name: 'name', label: 'label', value: 'value'))
```

Comment:
I fix a typo, and now I have only a problem with the lookbook. But I need to open PR in lookbook gem. 
<img width="1482" alt="CleanShot 2023-06-04 at 00 00 22@2x" src="https://github.com/mateuszbialowas/ogloradar/assets/58574619/815b0466-9d16-4563-8462-c8adf96dd152">

Reply:
![CleanShot 2023-06-04 at 00 00 26@2x](https://github.com/mateuszbialowas/ogloradar/assets/58574619/75edf5e5-a3c2-4fcc-9ff3-4bf87e0376dc)

Reply:
https://github.com/mateuszbialowas/ogloradar/pull/20